daemon/src/auto_rollover.rs

```diff
@@ -143,14 +143,17 @@ where
 #[async_trait]
 impl<O> xtra::Actor for Actor<O>
 where
-    O: 'static,
-    Self: xtra::Handler<AutoRollover>,
+    O: xtra::Handler<oracle::GetAnnouncement> + 'static,
```

Comment:
❓ What's this change about?

Reply:
Uhm I got weird compile-errors without it. Honestly don't understand why :see_no_evil: